assets/src/js/frontend/give-ajax.js

```diff
@@ -261,6 +261,8 @@ jQuery( document ).ready( function( $ ) {
                     this_form.find( '#give_purchase_submit input[type="submit"].give-submit' ).before( data );
                 }
 
+                this_form.closest( 'body' )[0].scrollIntoView({ behavior: 'smooth' });
```

Comment:
I'm assuming that the closest `body` would be the top of the iframe. but not all templates use the iframe - for example the Legacy Template is just HTML on the page.

Instead, maybe we could scroll to the Form Header, if in use? Or maybe there is another scroll to top example that you could reference.

Reply:
![CleanShot 2022-10-19 at 15 03 04](https://user-images.githubusercontent.com/3921017/196769716-3778c5f6-8ce1-4223-a667-15e3d0287f22.gif)
*Legacy Template*

![CleanShot 2022-10-19 at 15 03 54](https://user-images.githubusercontent.com/3921017/196769931-c6640f27-e37e-466a-9ffe-4f38f8c56100.gif)
*Classic Template*